app/src/main/java/com/bootcamp_android/parking_app/viewmodel/lot_detail/ReservationsViewModel.kt

```diff
@@ -8,25 +10,22 @@ import com.bootcamp_android.domain.DeleteReservationUseCase
 import com.bootcamp_android.domain.model.Reservation
 import com.bootcamp_android.domain.util.Result
 import com.bootcamp_android.domain.util.Utils
-import com.bootcamp_android.domain.util.Utils.parkingId
 import kotlinx.coroutines.launch
 
+class ReservationsViewModel(
+    val addReservationUseCase: AddReservationUseCase,val deleteReservationUseCase: DeleteReservationUseCase
+) : ViewModel() {
 
-class ReservationsViewModel(val addReservationUseCase: AddReservationUseCase, val deleteReservationUseCase: DeleteReservationUseCase): ViewModel() {
-
-    private val parkingId = Utils.parkingId
-    private val mutableSuccessfulAdd = MutableLiveData<Boolean>()
-    private val mutableSuccessfulDelete = MutableLiveData<Boolean>()
-    val successfulDelete: LiveData<Boolean> = mutableSuccessfulDelete
 
-    val successfulAdd: LiveData<Boolean> = mutableSuccessfulAdd
+    val mutableSuccessfulAdd = MutableLiveData<Boolean>()
```

Comment:
You should use a private mutablelivedata and a public livedata, just like you did in LotViewModel